### PR TITLE
Mongo service

### PR DESCRIPTION
**Pull request by DanielSchiavini**

### CODE REVIEW

File: mongoOperator/services/MongoService.py [hunk 1]
```diff
@@ -1,21 +1,89 @@
...
+
+    def __init__(self, kubernetes_service: KubernetesService):
+        self.kubernetes_service = kubernetes_service
+
+    def initializeReplicaSet(self, cluster_object: V1MongoClusterConfiguration) -> None:
+        name = cluster_object.metadata.name
+        namespace = cluster_object.metadata.namespace
+        replicas = cluster_object.spec.mongodb.replicas
+
+        pod_name = "{}-0".format(name)
```
Comment (ChrisTerBeke) [inline mongoOperator/services/MongoService.py]: Doesn't it need to be initialized in all instances that should be part of the replica set? Or will Mongo handle that when you create a connection string that points to all one of which is the 'master'?

Comment (DanielSchiavini) [reply mongoOperator/services/MongoService.py]: > Run rs.initiate() on just one and only one mongod instance for the replica set.

https://docs.mongodb.com/manual/reference/method/rs.initiate/

Comment (ChrisTerBeke) [reply mongoOperator/services/MongoService.py]: Cewl cewl

Comment (DanielSchiavini) [reply mongoOperator/services/MongoService.py]: haha I had to check :laughing:

File: mongoOperator/MongoOperator.py [hunk 2]
```diff
@@ -2,57 +2,39 @@
...
+        """
+        checker = ClusterChecker()
         try:
-            while True:
-                # Run the short-lived manager threads every few seconds but only allow a single instance per type.
-                for thread in self._manager_threads:
-                    if not thread.ident:
-                        thread.start()
-                sleep(self._check_seconds)
+            for _ in range(100):  # TODO: return this to: while True:
```
Comment (ChrisTerBeke) [inline mongoOperator/MongoOperator.py:27]: Can this be done now?

**Review by ChrisTerBeke — APPROVED** [review]

**Pull request merged**